README.md

```diff
@@ -20,6 +20,16 @@ nodejs:
   configmap:
     VAR_A: VALUE_A
     VAR_B: VALUE_B
+  keyVaults:
+    cmc:
+      resourceGroup: cmc
+      secrets:
+        - smoke-test-citizen-username
+        - smoke-test-ushmmer-password
```

Comment:
```suggestion
        - smoke-test-user-password
```

Reply:
thi is what is is in the keyvault I copied it directly :/